Tools/TestWebKitAPI/Tests/WebKitCocoa/ElementTargetingTests.mm

```diff
@@ -153,4 +167,39 @@ @implementation _WKTargetedElementInfo (TestingAdditions)
     EXPECT_TRUE([nextThreeSelectors containsObject:@".absolute.bottom-right"]);
 }
 
+TEST(ElementTargeting, AdjustVisibilityForUnparentedElement)
+{
+    auto webViewFrame = CGRectMake(0, 0, 800, 600);
+
+    RetainPtr webView = adoptNS([[TestWKWebView alloc] initWithFrame:webViewFrame]);
+    [webView synchronouslyLoadTestPageNamed:@"element-targeting-2"];
+
+#if PLATFORM(IOS_FAMILY)
+    auto window = adoptNS([[UIWindow alloc] initWithFrame:[webView frame]]);
+    [window addSubview:webView.get()];
+#endif
```

Comment:
I think you can use `-[TestWKWebView initWithFrame:configuration:addToWindow:]` rather than creating a window here.

Reply:
Good catch!